src/test/scala/beam/router/TollRoutingSpec.scala

```diff
@@ -111,7 +111,8 @@ class TollRoutingSpec
             None,
             None
           )
-        )
+        ),
+        triggerId = 0
```

Comment:
I cannot come up with a method that would help here. So I just added the triggerId=0 everywhere.

Reply:
I was thinking along the lines of:

```
object TriggerHelper{
  def generateTriggerId = 0
}
```

Of course you could then add ways to track it through correlationId or something, but that might be overkill if it works as is for now. But at least methodizing it would result in a central location to change instead of everywhere. Again, this is already done - so probably not worth doing now - maybe if it becomes cumbersome in the future, though :)